OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseUnitWebViewFragment.java

```diff
@@ -114,6 +129,9 @@ public void onPageLoadError(WebView view, WebResourceRequest request, WebResourc
 
             @Override
             public void onPageLoadProgressChanged(WebView webView, int progress) {
+                if (progress > 30) {
+                    evaluateJavascriptForiFrame();
```

Comment:
This function will be called every time progress is greater than 30. :fire: 
Maybe we should just display the banner regardless of the loading state.

Reply:
Code Updated.